packages/react/src/components/Footer/__tests__/LegalNav.test.js

```diff
@@ -37,7 +37,7 @@ describe('<LegalNav />', () => {
 
     expect(legalNav.exists('.bx--legal-nav')).toBeTruthy();
     expect(legalNav.find('.bx--legal-nav__list-item')).toHaveLength(
-      FOOTER_NAV_DATA.length - 1
+      FOOTER_NAV_DATA.length
```

Comment:
I understand why you removed the `- 1` here, but I wonder if we should consider adding it back. The reason it was there is that this mock data remains to have the same amount of items, but because the data is malformed the component doesn't render the malformed item leaving it within this scenario one less than the `data.length`.